colcon_cmake/task/cmake/__init__.py

```diff
@@ -206,6 +207,20 @@ def get_project_file(path, target):
     return project_file
 
 
+@functools.lru_cache(maxsize=1)
+def get_msbuild_version():
+    """
+    Get the version of msbuild or throw an exception.
```

Comment:
The docblock should contain more information what kind of exceptions can be raised. This should use the `:raises ExceptionType: ...` style.